src/helpers/figures/charts/chart_common_line_scatter.ts

```diff
@@ -246,10 +245,12 @@ export function createLineOrScatterChartRuntime(
     config.options.scales!.x!.ticks!.callback = (value) =>
       formatValue(value, { format: labelFormat, locale });
     config.options.plugins!.tooltip!.callbacks!.title = (tooltipItem) => {
-      return formatValue(tooltipItem[0].parsed.x || tooltipItem[0].label, {
-        locale,
-        format: labelFormat,
-      });
+      return tooltipItem[0].datasetIndex! < dataSetsCount
```

Comment:
same as my first comment in the side panl, this will easily break if we change the order of datasets